core/src/services/routing.js

```diff
@@ -405,3 +409,27 @@ export const handleRouteClick = (node, windowElem = window) => {
   const route = buildRoute(node, `/${node.pathSegment}`);
   navigateTo(route, windowElem);
 };
+
+export const getModifiedHash = s => s.newURL.split('#/')[1];
```

Comment:
This function is very specific and tightly coupled to the `hashchange` event listener. I would rather keep it inside the listener.

Reply:
Agree, done.